cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,28 @@
+import time
+import psycopg2
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from psycopg2 import OperationalError

```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)